Fixed grammar in "Awaiting for signals" end note (#5586)
godotengine/godot-docs: tutorials/scripting/gdscript/gdscript_basics.rst

        await get_signal()
        print("Button was pressed")

> .. note:: Unlike ``yield`` in previous Godot versions, you cannot obtain the function state object. This is done to ensure type safety.
> A function cannot say that it returns an ``int`` while it actually returns a function state object

Reviewer: ```suggestion
.. note:: Unlike ``yield`` in previous Godot versions, you cannot obtain the function state object.
          This is done to ensure type safety.
          With this type safety in place, a function cannot say that it returns an ``int`` while it actually returns a function state object
```

> .. note:: Unlike ``yield`` in previous Godot versions, you cannot obtain the function state object. This is in the spirit of
>           type-safety, because a function cannot say that it returns an ``int`` but actually return a function state object

Reviewer: ```suggestion
.. note:: Unlike ``yield`` in previous Godot versions, you cannot obtain the function state object. This is done to ensure type safety.
A function cannot say that it returns an ``int`` while it actually returns a function state object
```
